### DISCUSSION TITLE
Warning	CS0414 The field 'ColorSpectrum._isPointerOver' is assigned but its value is never used Avalonia.Controls.ColorPicker

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#8306] [Q&A] [answered]
https://github.com/AvaloniaUI/Avalonia/blob/89d0f1d8dca9dbc35a2c43e1fbb21f8fa6691895/src/Avalonia.Controls.ColorPicker/ColorSpectrum/ColorSpectrum.cs#L47

@robloo can be removed?

**Comment by robloo** [comment, marked as answer]
@workgroupengineering Yea, that warning is on my radar. The field is remaining from the WinUI code and was just never used. I left it in just in case. However, the ColorPicker is almost entirely complete functionality-wise and it still isn't needed. So I'll go ahead and fix this warning in PR #8215. You don't need to include it your other warning fixes.

**Reply by robloo** [reply]
Done in PR #8215
